src/Kreta/TaskManager/Domain/Model/Organization/OrganizationParticipant.php

```diff
@@ -14,13 +14,13 @@
 
 namespace Kreta\TaskManager\Domain\Model\Organization;
 
+use Kreta\SharedKernel\Domain\Model\Identity\EmailAddress;
+use Kreta\SharedKernel\Domain\Model\Identity\Username;
+
 class OrganizationParticipant extends Participant
 {
-    public function __construct(
-        OrganizationParticipantId $id,
-        OrganizationParticipantEmail $email,
-        OrganizationParticipantUsername $username
-    ) {
+    public function __construct(OrganizationParticipantId $id, EmailAddress $email, Username $username)
```

Comment:
It sounds weird to me to have email address and username inside domain each time a user is added as a participant. This requires keeping it on sync with user info all times